src/NuGet/Microsoft.Orleans.OrleansTelemetryConsumers.Counters.nuspec

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
+  <metadata>
+    <id>Microsoft.Orleans.OrleansTelemetryConsumers.Counters</id>
```

Comment:
Naive question, but do you think there is still value on having a separate nuget package for the perf counters installer? Maybe they can still live in different DLLs, but potentially shipped in a shared nuget?